app/src/main/java/nu/yona/app/ui/dashboard/DayActivityDetailFragment.java

```diff
@@ -326,131 +336,153 @@ private void setDayActivityDetails()
 		EmbeddedYonaActivity embeddedYonaActivity = YonaApplication.getEventChangeManager().getDataState().getEmbeddedDayActivity();
 		if (embeddedYonaActivity != null && embeddedYonaActivity.getDayActivityList() != null && embeddedYonaActivity.getDayActivityList().size() > 0)
 		{
+			setUpDayActivityListFromEmbeddedYonaActivity(embeddedYonaActivity);
+			fetchCommentsForCurrentIndex();
+		}
+		else
+		{
+			goBack();
+		}
+		setDayDetailTitleAndIcon();
+	}
 
+	private void setUpDayActivityListFromEmbeddedYonaActivity(EmbeddedYonaActivity embeddedYonaActivity)
+	{
+		try
```

Comment:
This try/catch can be omitted, as there are no checked exceptions thrown from within this block that need to be caught.